src/display/api.js

```diff
@@ -91,7 +91,7 @@ const DefaultStandardFontDataFactory =
  */
 
 /**
- * @type IPDFStreamFactory
+ * @type {IPDFStreamFactory}
```

Comment:
This is unrelated to the other changes, but I happened to notice this typo and thought that a separate commit/PR for *only* this changed was excessive :-)